docs/joint_models_estimating_parameters.rst

```diff
@@ -10,7 +10,282 @@ Creating a global hierarchical model
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 
-Implementing a new univariate distribution
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Implementing new statistical distributions in virocon: Example of the generalized gamma distribution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+When implementing a new distribution, we recommend orientating oneself on an already existing distribution in virocon.
+You can use a virocon distribution as a template to implement a new distribution. The following sections describe how
+the generalized gamma distribution was implemented.
 
+**1. Clarify the mathematics of the distribution**:
+
+In the following, we implement the 3-parameter generalized gamma distribution as recommended by Ochi (1992) [1]_.
+It's probability density function (pdf) is defined as:
+
+:math:`f(x)= \frac{c}{\Gamma(m)}\lambda^{cm}x^{cm-1} \exp\left[- (\lambda x)^{c} \right]`
+
+To implement the generalized gamma distribution, we make use of the functionality of scipy’s implementation of the
+distribution. Scipy’s implementation is based on the scientific paper of Stacy (1962) [2]_. An overview of the
+generalized gamma distribution is given in [3]_.
+Scipy uses the following pdf:
+
+:math:`f(x)=  \frac{k(x-a)^{kc-1}}{b^{kc}\Gamma(c)} \exp \bigg[- \bigg(\frac{x-a}{b}\bigg)^{k}\bigg]`
+
+Stacy’s generalized gamma distribution involves 4 parameters. Two shape parameters (k, c) , one scale (b) and one
+location parameter (a). Note that Ochi's and Stacy's formulas for the generalized gamma distribution
+differ. Hence, to use scipy’s functionality, we must convert the individual parameters. Comparing the parameters,
+it is seen, that:
+
+1. c=k ,
+2. m=c ,
+3. a=0 , and
+4. λ=1/b
+
+Here, the two shape parameters can be implemented using scipy’s shape parameters. The location parameter will be set to
+the fixed value zero and the scale parameter needs to be converted and inverted.
+
+
+**2. Define the characteristics of the distribution and implement it step by step:**
+
+To implement a new distribution, the user should always subclass the Distribution class in distributions.py.
+
+This init method is called when an object of the class GammaDistribution is created. When a new gamma distribution
+object is created, we want to make sure that the attributes of the distribution are passed. Therefore, we build a custom
+initial state, where all parameters of the distribution are initialized with a default value.
```

Comment:
"custom initial state" sounds not familiar to me. Write "custom constructor" instead?